test/browser.js

```diff
@@ -18,7 +19,6 @@ idb.deleteDatabase('ipfs')
 idb.deleteDatabase('ipfs/blocks')
```

Comment:
we probably should expose a function in ipfs-repo to clean/reset a repo, as we keep doing this and it is not intuitive to everyone.